src/pydvl/influence/influence_calculator.py

```diff
@@ -0,0 +1,554 @@
+from abc import ABC, abstractmethod
+from math import prod
+from typing import (
+    Callable,
+    Generator,
+    Generic,
+    Iterable,
+    List,
+    Optional,
+    Tuple,
+    TypeVar,
+)
+
+import distributed
+from dask import array as da
+from dask import delayed
+from numpy.typing import NDArray
+
+from .base_influence_model import (
+    InfluenceFunctionModel,
+    InfluenceType,
+    TensorType,
+    UnSupportedInfluenceTypeException,
+)
+
+__all__ = ["DaskInfluenceCalculator"]
+
+
+class DimensionChunksException(ValueError):
```

Comment:
Isn't this more natural?
```suggestion
class ChunksDimensionsException(ValueError):
```

Reply:
Renamed it to InvalidDimensionChunksError